OMCompiler/Compiler/boot/LoadCompilerSources.mos

```diff
@@ -452,6 +452,7 @@ if true then /* Suppress output */
     "../SimCode/HpcOmSimCodeMain.mo",
     "../SimCode/SerializeInitXML.mo",
     "../SimCode/SerializeModelInfo.mo",
+    "../SimCode/SerializeSparsityPattern.mo",
```

Comment:
Does boot need this?

Reply:
I thought so. Pretty much everything is listed here. But randomly removing stuff doesn't seem to make a difference. Perhaps we should remove everything that is not needed here.